style.css

```diff
@@ -0,0 +1,4 @@
+h3 {
+  color: rebeccapurple;
+  font-size: 50px;
+}
```

Comment:
- Kindly check if this file belongs to this project. If needed, please create an `<link>` on index.html to it. Otherwise, I think it is better to remove this file to the cleanest code.